### PR TITLE
ISPN-13207 Jakarta Jars

### PR DESCRIPTION
**Pull request by wburns**
https://issues.redhat.com/browse/ISPN-13207

### CODE REVIEW

File: client/hotrod-client/pom.xml [hunk 1]
```diff
@@ -571,6 +571,22 @@
                </execution>
             </executions>
          </plugin>
+         <plugin>
+            <groupId>org.eclipse.transformer</groupId>
+            <artifactId>org.eclipse.transformer.maven</artifactId>
+            <executions>
```
Comment (pruivo) [inline client/hotrod-client/pom.xml:577]: I would move the plugin execution to parent so you don't have to copy it in 3 different modules.

Comment (wburns) [reply client/hotrod-client/pom.xml:577]: Yeah, that is no bueno, creates the jar even if there are no classes with javax imports.

Comment (wburns) [reply client/hotrod-client/pom.xml:577]: Aha, thanks. I will do that.

Comment (wburns) [reply client/hotrod-client/pom.xml]: I have updated it to use pluginDependencies which reduces the other poms to a few lines instead. I think this is much better.

**Comment by pruivo** [comment]
merged! thanks @wburns !

**Pull request merged**